packages/api/src/command/medical/patient/delete-patient.ts

```diff
@@ -33,33 +36,28 @@ export const deletePatient = async (
     try {
       // TODO: #393 move to declarative, event-based integration
       // Synchronous bc it needs to run after the Patient is deleted (it needs patient data from the DB)
-      const results = await Promise.allSettled([
-        cwCommands.patient.remove(patient, facilityId),
-        fhirApi.deleteResource("Patient", patient.id),
-        cqCommands.patient.remove(patient),
+      await Promise.allSettled([
+        cwCommands.patient.remove(patient, facilityId).catch(err => {
+          if (err.response?.status !== 404) throw err;
```

Comment:
For this throw to reach the outer try/catch we need to move from `Promise.allSettled` to `Promise.all`, right?

Reply:
Oh thats right